graph/list.go

```diff
@@ -18,18 +18,24 @@ var acceptedImageFilterTags = map[string]struct{}{
 	"label":    {},
 }
 
+// ImagesConfig defines the criteria to obtain a list of images.
 type ImagesConfig struct {
+	//Filters is supported list of filters used to get list of images
 	Filters string
-	Filter  string
-	All     bool
+	//Filter the list of images by name
+	Filter string
+	//All is true all the images will be returned in the list
```

Comment:
doesn't read correctly, should be something like "All determines whether to return all images in the list"